app/assets/src/components/common/ProjectCreationForm.jsx

```diff
@@ -60,7 +62,21 @@ class ProjectCreationForm extends React.Component {
     }
   };
 
+  toggleInfo = () => {
+    const { showInfo } = this.state;
+
+    logAnalyticsEvent("ProjectCreationForm_more-info-toggle_clicked", {
+      showInfo,
```

Comment:
nit: this might be misleading when `showInfo` `true` means close the box

Reply:
Right, I should do this in a callback when setting the new state. Thanks for catching!